src/snowflake/snowpark/modin/pandas/_index.py

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+import pandas as native_pd
+
+
+class Index:
```

Comment:
We should call this file `index.py` not `_index.py` since it is user-facing (i.e. users can interact with the Index class)

Reply:
If i call it `index.py`, I have some issues with the way the imports are formatted and it gives me a circular import issue, that's why it is named the way it is rn